test/hack/cleanup/main.go

```diff
@@ -112,7 +119,7 @@ func (i *instance) Type() string {
 	return "Instances"
 }
 
-func (i *instance) Get(ctx context.Context, expirationTime time.Time) (ids []string, err error) {
+func (i *instance) Get(ctx context.Context, _ string, expirationTime time.Time) (ids []string, err error) {
```

Comment:
We should be using the cluster-name here, right? Cleanup any instances that have the cluster name and provisioner name? Also, I think that this change needs to be updated to use the new beta label alongside the alpha label